src/StackExchange.Redis/RedisLiterals.cs

```diff
@@ -123,7 +123,9 @@ public static readonly RedisValue
             pubsub = "pubsub",
             replication = "replication",
             server = "server",
+            master = "master",
             slave = "slave",
+            sentinel = "sentinel",
```

Comment:
nit: alpha order please!

Reply:
No problem. They didn't appear to be in strict alpha order previously so I just grouped like ones together.